runtime/component.py

```diff
@@ -309,12 +307,17 @@ def createContainer(self, client):
 
     self.logger.debug('Starting container for component %s with command %s', self.getName(),
                       command)
-                      
+
     container = client.create_container(self.config.getFullImage(), command,
                                         user=self.config.getUser(),
                                         volumes=self.config.getVolumes(),
-                                        ports=[str(p) for p in self.config.getContainerPorts()],
-                                        environment=self.calculateEnvForComponent())
+                                        environment=self.calculateEnvForComponent(),
+					host_config=client.create_host_config(restart_policy=self.config.getRestartPolicy(),
+                                                                              volumes_from=self.config.volumes_from,
+                                                                              binds={ b.external: { "bind": b.volume } for b in self.config.bindings },
```

Comment:
Line is quite long; maybe breakout the dict construction into its own variable above?